dirsrvtests/tests/suites/import/regression_test.py

```diff
@@ -434,7 +434,59 @@ def test_large_ldif2db_ancestorid_index_creation(topo, _set_mdb_map_size):
         # The time for the ancestorid index creation should be less than 10s for an offline import of an ldif file with 100000 entries / 5 entries per node
         # Should be adjusted if these numbers are modified in the test
         assert etime <= 10
-    
+
+
+def create_backend_and_import(instance, ldif_file, suffix, backend):
+    log.info(f'Add suffix:{suffix} and backend: {backend}...')
+    backends = Backends(instance)
+    backends.create(properties={'nsslapd-suffix': suffix, 'name': backend})
+    props = {'numUsers': 10000, 'nodeLimit': 5, 'suffix': suffix}
+
+    log.info(f'Create a large nested ldif file using dbgen : {ldif_file}')
+    dbgen_nested_ldif(instance, ldif_file, props)
+
+    log.info('Stop the server and run offline import...')
+    instance.stop()
+    log.info('Measure the import time for the ldif file...')
+    start = time.time()
+    assert instance.ldif2db(backend, None, None, None, ldif_file)
+    end = time.time()
+    instance.start()
+    return end - start
+
+
+@pytest.mark.xfail(not _check_disk_space(), reason="not enough disk space for lmdb map")
```

Comment:
Is this condition necessary. IIUC the problem only exist with bdb.

Reply:
Good catch!